src/app/Fake.Deploy/DeploymentHelper.fs

```diff
@@ -29,7 +29,7 @@ let getActiveReleases dir = !!(dir @@ deploymentRootDir @@ "**/active/*.nupkg")
 
 /// Retrieves the NuSpec information for the active release of the given app.
 let getActiveReleaseFor dir (app : string) = 
-    !!(dir @@ deploymentRootDir @@ app @@ "/active/*.nupkg")
+    !!(dir @@ deploymentRootDir @@ app @@ "active/*.nupkg")
```

Comment:
Without that change several of the DeploymentHelper tests failed.  With the new combinePath using just a straight call to Path.Combine, app @@ "/active/*.nupkg" just returns "/active/*.nupkg".  And the tests will fail as there isn't anything on the filesystem at /active/whatever.  The leading forward slash implies its a root directory, and Path.Combine says if path2 is a root, just return path2. 

In effect, saying Path.Combine("something", "/active/*.nupkg") is the same as saying Path.Combine("something", "C:\\active\\*.nupkg") and right or not, that is supposed to return C:\active\*.nupkg.    There is no need to add the separator character to the beginning of path2 (the whole point of Path.Combine is to add that for you), and on linux and very likely osx, its also true that that separator can also be interpreted as a root directory indicator.  Its true that its a bit confusing when thinking of @@ as +, but the original stripping of the leading / caused things to work differently on windows and linux.

The original issue I raised was presented as a question because I was unsure about fixing it as I expected that such a change as this would break more than it did.  I'm guessing most everyone uses @@ as if it were a + operator, and don't fully realize its a Path.Combine under the hood and what that means, but it did cause different behavior cross platforms.